bindings/gumjs/gumscript-runtime-objc.js

```diff
@@ -443,7 +486,22 @@
                                 }
                             }
                             return cachedProtocols;
+                        case "$methods":
+                            if (cachedNativeMethodNames === null) {
+                                cachedNativeMethodNames = [];
+
+                                // Fill cachedMethodNames
+                                this.keys();
+
+                                cachedNativeMethodNames = Object.keys(cachedMethods).filter(m => m.match(/^(\+|-)/));
+                            }
+                            return cachedNativeMethodNames;
                         default:
+                            if (handle.isNull()) {
+                                return function() {
+                                    return 0;
```

Comment:
Only `ObjC.nil` will give you the ability to have a chain of calls that still "work", but as you said, it's still quite broken. Maybe we shouldn't bother with `ObjC.nil` and just expect you to pass `NULL` for such arguments?